CleverPush/Source/CPChatView.m

```diff
@@ -26,6 +26,13 @@ @implementation CPChatView
 
 #pragma mark - Load chat with subscription id
 - (void)loadChat {
+    BOOL isSubscriptionChanged = [CleverPush getSubscriptionChanged];
```

Comment:
Can we do the detection if subscription ID changed inside CPChatView, instead of in CleverPush.m subscribed handler?
It would be better because we only need this information in the CPChatView Class.

We already have the `subscriptionId` variable in CPChatView and can compare with `getSubscriptionId` inside loadChat

Reply:
@gbhrdt , We are comparing the old value with the new value here, and after that, the new value has been updated with the preference key.
So I think that if we get the subscriptionId key into the CPChatview class, we will get only the updated key and not the old one as well..